src/Compilers/CSharp/Portable/Binder/Semantics/BestTypeInferrer.cs

```diff
@@ -13,62 +13,22 @@ internal static class BestTypeInferrer
         public static NullableAnnotation GetNullableAnnotation(TypeSymbol bestType, ArrayBuilder<TypeSymbolWithAnnotations> types)
         {
             bool bestTypeIsPossiblyNullableReferenceTypeTypeParameter = bestType.IsPossiblyNullableReferenceTypeTypeParameter();
```

Comment:
>bestTypeIsPossiblyNullableReferenceTypeTypeParameter [](start = 17, length = 52)

It looks like this value and ```bestType``` are no longer used, however it feels like the result should be affected by them in one way or the other because it looks like we are inferring annotations for the ```bestType``` and it defines what annotations are speakable.